src/request/src/request.js

```diff
@@ -177,6 +180,7 @@ export default class Request {
 
   toPlainObject() {
     return {
+      id: this.id,
```

Comment:
This is the related [JIRA ticket](https://kinvey.atlassian.net/browse/MLIBZ-2025).